solr/core/src/java/org/apache/solr/cluster/events/ClusterEventProducer.java

```diff
@@ -26,7 +26,7 @@
 public interface ClusterEventProducer extends ClusterSingleton, Closeable {
 
   /** Unique name for the registration of a plugin-based implementation. */
-  String PLUGIN_NAME = "cluster-event-producer";
+  String PLUGIN_NAME = ".cluster-event-producer";
```

Comment:
I don't understand why "." was added, therefore I suggest adding a comment explaining it.

Reply:
There was a Slack discussion on the naming scheme for pre-defined plugins (on #solr-plugins) and the overall sentiment was to use a dot prefix for such names. It's a convention not a requirement - and it will be documented in SOLR-15022.